### PR TITLE
DRILL-6338: Do not skip license maven plugin when formatting generate…

### PR DESCRIPTION
**Pull request by ilooner**
…d protobuf files.

And fixed malformed .travis.yml

### CODE REVIEW

**Comment by ilooner** [comment]
@vrozov @arina-ielchiieva Please review.

File: protocol/pom.xml [hunk 1]
```diff
@@ -149,8 +152,12 @@
           <plugin>
             <groupId>com.mycila</groupId>
             <artifactId>license-maven-plugin</artifactId>
+            <configuration>
```
Comment (vrozov) [inline protocol/pom.xml]: Do no force `<skip>`. It should be already `false`.

Comment (ilooner) [reply protocol/pom.xml]: Thanks for catching. Removed, verified that `mvn process-sources -P proto-compile` and `mvn clean install -P proto-compile` generated code with the headers and that `mvn process-sources -P proto-compile -Dlicense.skip=true` generates sources without license headers.

**Comment by arina-ielchiieva** [comment]
LGTM.

**Pull request closed without merging**